app/controllers/api/account.php

```diff
@@ -215,6 +215,9 @@
             $total = $dbForProject->count('users', max: APP_LIMIT_USERS);
 
             if ($total >= $limit) {
+                if ('console' === $project->getId()) {
+                    throw new Exception(Exception::USER_CONSOLE_COUNT_EXCEEDED);
+                }
```

Comment:
There are a lot more places where we check the auth limit 😬 but i guess it's not really used in self hosted.